core/jobs/batch_jobs/exploration_migration_jobs.py

```diff
@@ -80,11 +80,44 @@ def _migrate_exploration(
 
         return result.Ok((exp_id, exploration))
 
+    @staticmethod
+    def _migrate_exploration_summary(
+        exp_id: str, exp_summary_model: exp_models.ExpSummaryModel
+    ) -> result.Result[
+        Tuple[str, exp_domain.ExplorationSummary], Tuple[str, Exception]
+    ]:
+        """Migrates exploration summary and transforms exploration summary model
+        into exploration summary object.
+
+        Args:
+            exp_id: str. The id of the exploration.
+            exp_summary_model: ExpSummaryModel. The exploration
+                model to migrate.
+
+        Returns:
+            Result((str, ExplorationSummary), (str, Exception)). Result
+            containing tuple that consists of exploration ID and either
+            exploration object or Exception. Exploration object is
+            returned when the migration was successful and Exception
+            is returned otherwise.
+        """
+        try:
+            exploration_summary = (
+                exp_fetchers.get_exploration_summary_from_model(
+                    exp_summary_model))
```

Comment:
In exp_services.py `_compute_summary_of_exploration` we also do some other stuff with the summary, is it not needed here (not sure)?

Reply:
Do you suggest to copy all the things which we are doing in `_compute_summary_of_exploration` here?

Reply:
Not sure, I think we should mimic the classic exploration saving, but it might not be needed because we might know that these lines are not needed because xxx (for example if we only raise of some count when there is a new person that played the exploration we don't need to add it here, because here we only migrate so there will be no player count change).

Reply:
Did you verify this?